applications/asset_tracker/src/env_sensors/env_sensors.c

```diff
@@ -54,6 +57,21 @@ static struct env_sensor *env_sensors[] = {
 };
 
 static struct k_delayed_work env_sensors_poller;
+static env_sensors_data_ready_cb data_ready_cb;
+static u32_t data_send_interval_s = CONFIG_ENVIRONMENT_DATA_SEND_INTERVAL;
+static bool backoff_enabled;
+static bool initialized;
+
+static inline int submit_poll_work(const u32_t delay_s)
+{
+	return k_delayed_work_submit(&env_sensors_poller,
+			K_SECONDS((u32_t)delay_s));
```

Comment:
The arguments could be aligned here as well, but that's just personal preference, checkpatch doesn't mind :)